pkg/git/gitlab.go

```diff
@@ -0,0 +1,13 @@
+package git
+
+import (
+	"context"
+
+	"github.com/codefresh-io/cli-v2/pkg/log"
+)
+
+func VerifyGitLabTokenScope(ctx context.Context, token string) (bool, error) {
```

Comment:
If the function name includes the provider, I don't see a point in separating it to different files